content/workload-reduction-toolkit/evaluate-workload-measures/staff-workload-survey.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Staff workload survey
+layout: /markdown_resource.html.erb
+colour: green
+---
+
+# Evaluate impact using a staff workload survey
+
+<strong class="govuk-tag">Template</strong>
+
+Repeat the [survey](/workload-reduction-toolkit/identify-workload-issues/staff-workload-survey) in the
```

Comment:
I know this is copied from the prototype, but would be intrigued to get Lauren or Ashleigh's perspective on if this bit either requires a heading or should move out the box. It looks a bit weird to me without either.

Reply:
We'll add this in another PR.